backend/workspaces/views.py

```diff
@@ -526,49 +545,56 @@ def add_annotators(self, request,pk=None):
             workspace.save()
             if len(invalid_user_ids) == 0:
                 return Response({"message": "users added successfully"}, status=status.HTTP_200_OK)
-            elif len(invalid_user_ids)==len(user_ids):
+            elif len(invalid_user_ids) == len(user_ids):
                 return Response({"message": "No valid user_ids found"}, status=status.HTTP_400_BAD_REQUEST)
 
-            return Response({"message": f"users added partially! Invalid user_ids: {','.join(invalid_user_ids)}"}, status=status.HTTP_200_OK)
+            return Response(
+                {"message": f"users added partially! Invalid user_ids: {','.join(invalid_user_ids)}"},
+                status=status.HTTP_200_OK,
+            )
         except Workspace.DoesNotExist:
             return Response({"message": "Workspace not found"}, status=status.HTTP_404_NOT_FOUND)
         except ValueError:
             return Response({"message": "Server Error occured"}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
-    
     @swagger_auto_schema(
         method="post",
         request_body=openapi.Schema(
             type=openapi.TYPE_OBJECT,
-            properties={
-                "user_id": openapi.Schema(type=openapi.TYPE_STRING, format="email")
-            },
-            required=["user_id"]
+            properties={"user_id": openapi.Schema(type=openapi.TYPE_STRING, format="email")},
+            required=["user_id"],
         ),
         manual_parameters=[
             openapi.Parameter(
-                "id", openapi.IN_PATH,
+                "id",
+                openapi.IN_PATH,
                 description=("A unique integer identifying the workspace"),
                 type=openapi.TYPE_INTEGER,
-                required=True
+                required=True,
```

Comment:
Is this required to add a comma after last argument? Can you make sure it won't break anything?